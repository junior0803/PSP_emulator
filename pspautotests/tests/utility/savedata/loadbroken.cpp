#include "broken.h"

extern "C" int main(int argc, char **argv) {
	initDisplay();

	for (int i = 1; i <= 3; i++) {
		runBrokenTrials("auto load", SCE_UTILITY_SAVEDATA_TYPE_AUTOLOAD, i);
		runBrokenTrials("load", SCE_UTILITY_SAVEDATA_TYPE_LOAD, i);
		runBrokenTrials("list load", SCE_UTILITY_SAVEDATA_TYPE_LISTLOAD, i);
		runBrokenTrials("read data secure", SCE_UTILITY_SAVEDATA_TYPE_READDATASECURE, i);
		runBrokenTrials("read data", SCE_UTILITY_SAVEDATA_TYPE_READDATA, i);
	}

	return 0;
}