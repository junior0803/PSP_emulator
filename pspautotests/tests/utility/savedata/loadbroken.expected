[x] auto load(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:0): 00000000
[r] result: 00000000 => 80110309

[x] load(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:2): 00000000
[r] result: 00000000 => 80110309

[x] list load(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:4): 00000000
[r] result: 00000000 => 80110309

[x] read data secure(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:15): 00000000
[r] result: 00000000 => 80110329

[x] read data(broken 1):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] System:PARAM.SFO
[r] sceUtilitySavedataInitStart(mode:16): 00000000
[r] result: 00000000 => 80110329

[x] auto load(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:0): 00000000
[r] result: 00000000 => 80110306

[x] load(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:2): 00000000
[r] result: 00000000 => 80110306

[x] list load(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:4): 00000000
[r] result: 00000000 => 80110306

[x] read data secure(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:15): 00000000
[r] result: 00000000 => 80110326

[x] read data(broken 2):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:16): 00000000
[r] result: 00000000 => 80110326

[x] auto load(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:0): 00000000
[r] result: 00000000 => 80110306

[x] load(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:2): 00000000
[r] result: 00000000 => 80110306

[x] list load(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:4): 00000000
[r] result: 00000000 => 80110306

[x] read data secure(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:15): 00000000
[r] result: 00000000 => 80110326

[x] read data(broken 3):
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] result: 00000000 => 80110326
[r] Files:
[x] System:
[r] sceUtilitySavedataInitStart(mode:16): 00000000
[r] result: 00000000 => 80110326
